frontend/views/components/GroupWelcome.vue

```diff
@@ -86,11 +87,6 @@ export default ({
 
   .c-description {
     margin: 0 0 0.5rem;
-
-    @include phone {
-      position: absolute;
-      top: 5.5rem;
-    }
```

Comment:
@Silver-IT Can you explain these CSS changes?

Reply:
This commit shouldn't be here in this PR because it is for the first problem of #1117.
`Group picture and description are overlapped`